### DISCUSSION TITLE
Unknown problem somewhere in the system

### DISCUSSION THREAD

**Discussion by wazuhnoob** [wazuh/wazuh#14118] [General]
Hi,
After i restarted my wazuh-server, i started received these logs non stop (Unknown problem somewhere in the system),
wazuh opensearch-dashboards: {"type":"log","@timestamp":"2022-07-04T12:12:34Z","tags":["error","opensearch","data"],"pid":356,"message":"[version_conflict_engine_exception]: [search-telemetry:search-telemetry]: version conflict, required seqNo [486], primary term [4]. current document has seqNo [487] and primary term [

wazuh opensearch-dashboards: {"type":"error","@timestamp":"2022-07-04T12:12:34Z","tags":["connection","client","error"],"pid":356,"level":"error","error":{"message":"139638071605120:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1544:SSL alert number 46\n","name":"Error","stack":"Error: 139638071605120:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1544:SSL alert number 46\n"},"message":"139638071605120:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1544:SSL alert number 46\n"}

Understand that these will keep triggering email alerts, 
Please help. thank you.

**Comment by mdiego92** [comment]
Hello wazuhnoob,

Regarding the first log entry, It’s possible that the policy currently applied to your index isn’t the most up-to-date policy available. To see what policy is currently applied to your index, see [Explain index](https://opensearch.org/docs/latest/im-plugin/ism/api/#explain-index). To get the most up-to-date version of a policy, see [Get policy](https://opensearch.org/docs/latest/im-plugin/ism/api/#get-policy).

The second log entry is notifying issues when trying to connect with a self-signed cert. Could you please send us the output of the following command?

`curl https://127.0.0.1:9200/ -k -u kibanaserver:kibanaserver 
`

**Did you add the password of the kibanaserver user to the Wazuh Dashboard keystore?**
Please execute the command below and follow the instructions. You may find your old password in the /etc/kibana/kibana.yml configuration file.

`/usr/share/wazuh-dashboard/bin/opensearch-dashboards-keystore --allow-root add opensearch.password
`

Optional action - To change the default user, run the following command. You will need to change the password accordingly.

`/usr/share/wazuh-dashboard/bin/opensearch-dashboards-keystore --allow-root add opensearch.username
`

Please let us know if you have any remaining questions or doubts.

Thank you very much.
